db/large_data_handler.cc

```diff
@@ -46,17 +46,22 @@ large_data_handler::large_data_handler(uint64_t partition_threshold_bytes, uint6
         partition_threshold_bytes, row_threshold_bytes, cell_threshold_bytes, rows_count_threshold);
 }
 
-future<bool> large_data_handler::maybe_record_large_partitions(const sstables::sstable& sst, const sstables::key& key, uint64_t partition_size) {
+future<std::pair<bool, bool>> large_data_handler::maybe_record_large_partitions(const sstables::sstable& sst, const sstables::key& key, uint64_t partition_size, uint64_t rows_count) {
```

Comment:
Since we rely on the order of which status indicates what,
we better define a struct with named members so the order in the pair wouldn't matter.
For example:
```
struct large_partition_above_threshold {
    bool partition_size;
    bool too_many_rows;
};
```